### PR TITLE
CDAP-1368: Use v3 APIs in TestFrameworkTest, AppFabricTestHelper for lifecycle management

### PR DESCRIPTION
**Pull request by jwang47**
Depends on https://github.com/caskdata/cdap/pull/1567

https://issues.cask.co/browse/CDAP-1368
http://builds.cask.co/browse/CDAP-DUT1018

### CODE REVIEW

File: cdap-client-tests/src/test/java/co/cask/cdap/client/ScheduleClientTestRun.java [hunk 1]
```diff
@@ -1,5 +1,9 @@
 /*
+<<<<<<< HEAD
```
Comment (anwar6953) [inline cdap-client-tests/src/test/java/co/cask/cdap/client/ScheduleClientTestRun.java]: resolve

File: cdap-client/src/main/java/co/cask/cdap/client/ScheduleClient.java [hunk 2]
```diff
@@ -1,5 +1,9 @@
 /*
+<<<<<<< HEAD
```
Comment (anwar6953) [inline cdap-client/src/main/java/co/cask/cdap/client/ScheduleClient.java]: resolve

**Comment by anwar6953** [comment]
The design to namespace the ApplicationManager/ProgramManager looks good to me.
I think it'd be good to merge the PR this depends on, to see the code diff better.

**Comment by jwang47** [comment]
Reopened against release/2.8 branch: https://github.com/caskdata/cdap/pull/1603

**Pull request closed without merging**